test/controllers/insights_cloud/api/machine_telemetries_controller_test.rb

```diff
@@ -108,6 +108,7 @@ class MachineTelemetriesControllerTest < ActionController::TestCase
     context '#branch_info' do
       setup do
         User.current = User.find_by(login: 'secret_admin')
+        Setting[:allow_multiple_content_views] = true
```

Comment:
This is going to break downstream, unless we expose the setting via foreman_theme_satellite.

Reply:
Sorry, I am missing something. Is this setting introduced by the theme? I think I have seen it in Katello.

Reply:
additional context: This had been working (unintentionally) until Katello implemented https://projects.theforeman.org/issues/37657

Reply:
@jeremylenz Can you add a if condition or dry catch to skip the test if the setting is not there?